src/Components/DataHistoryExplorer/DataHistoryExplorer.tsx

```diff
@@ -496,6 +497,7 @@ const DataHistoryExplorer: React.FC<IDataHistoryExplorerProps> = (props) => {
         ) {
             updateConnectionAdapterData.callAdapter();
         }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
do we need to keep track of these to fix them later? or could you add a comment on the ones that we won't fix.

Reply:
I'm not planning to fix any of these actually. This one intentionally doesn't have any dependencies becaues it's supposed to only fire on mount.